source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -0,0 +1,163 @@
+=================
+Change a Document
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to change documents in a MongoDB
+collection using **update** and **replace** operations.
+
+Update operations modify the fields that you specify in one or more
+documents while leaving other fields and values unchanged. Replace
+operations remove all existing fields except for ``_id`` in
+one or more documents and substitute the deleted fields with new fields
+and values.
+
+.. _updateDocuments:
+
+Update
+------
+
+To update one document, use the ``UpdateOne()`` method, and to update
```

Comment:
Suggestion: Can the method be linked to the API docs? I think an inline link makes more sense than to have a "for more information" part at the end of the section.

Applies to all instances.

Reply:
I could forego monospacing the first instance of each method name to link it inline?

Reply:
Yea, I usually link the first/one mention of it and the rest remain monospaced.